client/src/components/StudentSearch.tsx

```diff
@@ -4,6 +4,7 @@ import { CourseService } from 'services/course';
 
 type Props = UserProps & {
   courseId: number;
+  hideWithoutMentor?: boolean;
```

Comment:
kind of confusing: title says we should hide students with mentor, although your prop says opposite. Am I missing smth?